packages/core/src/index.test.ts

```diff
@@ -12,3 +13,8 @@ test("basic arithmetic", () => {
   const g = interp(f);
   expect(g()).toBe(6);
 });
+
+// Printing a js fuction's rust IR to console
+const f = fn([Real, Real], (x, y) => add(x, y));
+const g = wasm.js2Rust(f.f.f);
+console.log(g);
```

Comment:
Since this is a test of internal functionality rather than the external API, could you make a new test file to put it in rather than `index.test.ts`? And then in the test could you have it maybe just check the result against some expected string, rather than directly calling `console.log` (so that it doesn't go in the console when the test passes)?